src/cart/swim/swim.c

```diff
@@ -1093,6 +1095,17 @@ swim_updates_parse(struct swim_context *ctx, swim_id_t from_id, swim_id_t id,
 	swim_ctx_lock(ctx);
 	ctx_state = swim_state_get(ctx);
 
+	rc = ctx->sc_ops->get_member_state(ctx, id, &id_state);
```

Comment:
This has been a constant source of confusion for me too, but would you mind confirming that we want to pass `id` here, not `from_id`? My reading of the `crt_swim_cli_cb` case seems to indicate that we should pass `from_id`.

Reply:
I have a confusion about this as well, there are two invocations about this function: [here](https://github.com/daos-stack/daos/blob/306af503ad839aa286a2de25f6b07a9dcd6fc17d/src/cart/crt_swim.c#L493) this node receives a request, and two ids are the same, therefore checking either one is fine.

The question is [here](https://github.com/daos-stack/daos/blob/306af503ad839aa286a2de25f6b07a9dcd6fc17d/src/cart/crt_swim.c#L639), should we use the to_id for IREQ instead?

I'm open to suggestion though, and I'd change that if `from_id` should be indeed used.